torch/csrc/jit/interned_strings.h

```diff
@@ -56,6 +56,13 @@ namespace torch { namespace jit {
   _(prim, FusedConcat)             \
   _(prim, FusedChunk)              \
   _(prim, NoneGenerator)           \
+  _(prim, MemoryFence)             \
+  _(prim, EntryWorld)              \
+  _(prim, ExitWorld)               \
+  _(prim, LoadWorld)               \
+  _(prim, StoreWorld)              \
+  _(prim, DummyWorld)              \
```

Comment:
prim::DummyWorld and prim::LoadWorld do the same thing: they load the 'current' world token, for whatever definition of current there is.

Reply:
Hm, the main reason for having `DummyWorld` is that the program will fault at runtime if that instruction is every executed by the interpreter.

Whereas every program should start with a `LoadWorld` instruction to set up the world token. This has the effect of forcing pass writers to deal with any `DummyWorld`s they are emitting.

Is there a better way to distinguish these two use-cases with a single op?

Reply:
I see, that makes sense as a reason for having two ops.